src/styles/blocks/cards.scss

```
@@ -0,0 +1,22 @@
+@import "./utils/functions";
+
+.cards__container {
```

```suggestion
.cards {
```
Make it work and use scss nesting